services/embeddings_demo/embeddings_demo.py

```diff
@@ -0,0 +1,37 @@
+import os
+from dotenv import load_dotenv
+from embeddings.demo.example import get_demo_vectorstore
```

Comment:
I'd really like a cleaner import here :thinking: 

How can we set up the package structure to support this?
```
from embeddings import demo

demo.add_docs()
demo.search()
```

That would be really neat. I was thinking about having a sub folder per file, which allows stores to have many supporting files and still give us a tidy folder structure. If that's in our way, maybe we have a top level entry point file (`example.py`), AND a subfolder to hide  away the messy stuff, if needed (`example/setup.py`, `example/data.json`)

We'd also have to do away with the `get_demo_vectorstore` and initialise the store when the module is loaded. Which, as I've asked elsewhere, I'm  not sure what the cost of that is.

If we really have to have an initialisation function, maybe we can just call it `connect`? Which I expect to give me a unique connection to the underlying database.

That would give me code like:

```
from embeddings import demo

store = demo.connect()
store.add_docs(...)
store.search(...)
```

Reply:
It seems a little unexpected to me to do something potentially costly/unwanted (wiping a database) in the background on just an import. If it's ok, I'll just rename it to connect() to make it clear that something will happen when you use the class instance.

Reply:
Yeah agreed!

`connect` should just give me a live connection to a database, which I can then do as I want. And it's fairly clear from that semantics that that's what I expect. So that's cool!

One note on this: in the example store, the `connect` function will also load data. Which is a little funky because if two  different services connect, they'll duplicate the data load.

Maybe just for the purposes of the demo, we can have a `connect()` and a `load_default_data()` function in the example store, and then the demo script will explicitly connect AND load the data?

This is also a  bit weird - we'd usually have some offline process to populate the store and not make it part of apollo like this. But it keeps the data loading logic in the store (which I like) and takes side-effects out of `connect` (which I like). So I think it's an improvement?

We'll have to look more closely at when and how the embedding stories get loaded. But we'll defer that here - we're going to bump into it pretty quickly on your mapper!

Reply:
I added a load_default_data() in the demo – does it look like what you meant?

Reply:
Perfect!